lib/widgets/picos_radio_select.dart

```diff
@@ -61,7 +63,23 @@ class _PicosRadioSelectState extends State<PicosRadioSelect> {
   @override
   Widget build(BuildContext context) {
     return Column(
-      children: _createItemList(),
+      children: <ListView>[
```

Comment:
Well two things:
1. The ListView makes the Column redundant. You should only use one or the other.
2. THe ListView makes the content scrollable even if it shouldn't scroll. Try to scroll within a PicosRadioSelect and you notice, the overscroll effect.